packages/context/README.md

```diff
@@ -1,5 +1,17 @@
 # @lit/context
 
+Make data available to entire component subtrees with the context protocol
```

Comment:
Optional: do we clarify what “THE context protocol” is elsewhere? Just to clarify it’s a community protocol and not just lit